vrx_gz/worlds/2023_practice/practice_2023_stationkeeping2_task.sdf

```diff
@@ -487,66 +463,21 @@
           key: "gain"
           value {
             type: DOUBLE
-            double_value: 0.8
-          }
-        }
-        params {
-          key: "model"
-          value {
-            type: STRING
-            string_value: "PMS"
-          }
-        }
-        params {
-          key: "number"
-          value {
-            type: INT32
-            int_value: 3
+            double_value: 2.0
           }
         }
         params {
           key: "period"
           value {
             type: DOUBLE
-            double_value: 5
-          }
-        }
-        params {
-          key: "phase"
-          value {
-            type: DOUBLE
-            double_value: 0
-          }
-        }
-        params {
-          key: "scale"
-          value {
-            type: DOUBLE
-            double_value: 1.1
-          }
-        }
-        params {
-          key: "size"
-          value {
-            type: VECTOR3D
-            vector3d_value {
-              x: 6000
-              y: 6000
-            }
+            double_value: 2.0
           }
         }
         params {
           key: "steepness"
           value {
             type: DOUBLE
-            double_value: 0
-          }
-        }
-        params {
-          key: "tau"
-          value {
-            type: DOUBLE
-            double_value: 2
+            double_value: 0.5
```

Comment:
Is this change on purpose? It's usually `0`.

Reply:
Apologies for the oversight, i forgot to reset the value back to 0 after conducting the tuning experiments, resulting in its unchanged state. I will update it in with a commit.